### PR TITLE
Fixed code Issue on tilexyz_services

### PR DESCRIPTION
**Pull request by Ajoke23**
All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [x] issues: fixes #xxxx
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

File: examples/topics/geo/tile_xyzservices.py [hunk 1]
```diff
@@ -1,10 +1,28 @@
-import xyzservices.providers as xyz
+''' A geographical plot that demonstrates how to use XYZ tile services to create an interactive map.
+The xyzservices provides map tiles,which are typically used to display maps visualizations
```
Comment (bryevdv) [inline examples/topics/geo/tile_xyzservices.py]: missing space
```suggestion
The xyzservices provides map tiles, which are typically used to display maps visualizations 
```
Also I am not sure we need to change anything here, but just being completely pedantic: many different services provide the actual map tiles. The role of the `xyzservices` project is to collect all of those different services and make all of their different URLs easy to find (kind of like a phone book).

File: examples/topics/geo/tile_xyzservices.py [hunk 2]
```diff
@@ -1,10 +1,28 @@
-import xyzservices.providers as xyz
+''' A geographical plot that demonstrates how to use XYZ tile services to create an interactive map.
+The xyzservices provides map tiles,which are typically used to display maps visualizations 
+and geographical location in web applications.The data used to create the plot is 
+sourced from public XYZ tile services.This example demonstrates how to create a plot in Bokeh
+with a map background using tile_providers.The chart shows a map of a specific geographical region, 
+which is defined by the range bounds supplied in web mercator coordinates.
```
Comment (bryevdv) [inline examples/topics/geo/tile_xyzservices.py]: missing spaces after sentence periods
```suggestion
and geographical location in web applications. The data used to create the plot is 
sourced from public XYZ tile services. This example demonstrates how to create a plot in Bokeh
with a map background using tile_providers. The chart shows a map of a specific geographical region, 
which is defined by the range bounds supplied in web mercator coordinates.
```

**Comment by bryevdv** [comment]
Hi @Ajoke23 What was the reason to remove the use of `xyzservices`? That is not actually something we should do. As a project we decided to offload the burden of curating tile services to `xyzservices` (since thy can do it much better than we could). We made `xyzservices` an explicit depencency of Bokeh, and this example was specifically to demonstrate how to use `xyzservices` (which is recommended for all users).

File: examples/topics/geo/tile_xyzservices.py [hunk 3]
```diff
@@ -1,10 +1,28 @@
...
+from bokeh.tile_providers import get_provider, Vendors
 
 # range bounds supplied in web mercator coordinates
 p = figure(x_range=(-2000000, 6000000), y_range=(-1000000, 7000000),
            x_axis_type="mercator", y_axis_type="mercator")
-p.add_tile(xyz.OpenStreetMap.Mapnik)
+
+# Retrieve the OpenStreetMap.Mapnik tile source from the tile providers
+tile_provider = get_provider(Vendors.CARTODBPOSITRON)
+p.add_tile(tile_provider)
```
Comment (bryevdv) [inline examples/topics/geo/tile_xyzservices.py]: All of the changes  here in the actual code should be reverted I think

**Review by bryevdv — CHANGES_REQUESTED** [review]
Please see comment about keeping `xyzservices`

**Comment by bryevdv** [comment]
@Ajoke23 this PR seems to overlap with https://github.com/bokeh/bokeh/pull/12987 should one of the be closed?

File: examples/topics/geo/tile_xyzservices.py (hunk 3 above)
Comment (Ajoke23) [reply examples/topics/geo/tile_xyzservices.py]: Alright but I keep getting error from my end about from the main code. Any help?

File: examples/topics/geo/tile_xyzservices.py (hunk 2 above)
Comment (Ajoke23) [reply examples/topics/geo/tile_xyzservices.py]: @bryevdv changes made. Kindly review

File: examples/topics/geo/tile_xyzservices.py (hunk 3 above)
Comment (Ajoke23) [reply examples/topics/geo/tile_xyzservices.py]: tileservices
error

Comment (Ajoke23) [reply examples/topics/geo/tile_xyzservices.py]: @bryevdv this is the error I was getting from the main code before making changes. What do you thin is the reason for getting such error?

Comment (bryevdv) [reply examples/topics/geo/tile_xyzservices.py]: @Ajoke23 I think your current env must have a rather old version of Bokeh installed. This example is featured in our users guide (second plot) and definitely works:

https://docs.bokeh.org/en/latest/docs/user_guide/topics/geo.html

**Comment by bryevdv** [comment]
Hi @Ajoke23 it looks like the changes to this file ended up in https://github.com/bokeh/bokeh/pull/12992 I am not sure if that was intended, but in an y case in order to keep things simple I am going to close this PR in favor of the newer one

superseded by https://github.com/bokeh/bokeh/pull/12992

**Pull request closed without merging**